cdn/utils/checksum.go

```diff
@@ -0,0 +1,14 @@
+package utils
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+)
+
+// ComputeChecksum computes a combined checksum for multiple files.
+func ComputeChecksum(file []byte) (string, error) {
+	hash := sha256.New()
+	hash.Write(file)
+
+	return hex.EncodeToString(hash.Sum(nil)), nil
+}
```

Comment:
Does this handle `nil` files? If it does, I suggest removing `error` from signature to return just `string`